kore/src/Kore/Builtin/AssociativeCommutative.hs

```diff
@@ -0,0 +1,1451 @@
+{- |
```

Comment:
Maybe use the new guidelines for Module headers. See, e.g., https://github.com/kframework/kore/blob/master/kore/src/Kore/Repl.hs

Reply:
Done